### ISSUE TITLE
metal ipmi command line weirdness

### ISSUE THREAD

**Issue by ste78** [alces-software/metalware#102]
[root@deploy ~]# metal ipmi -k 'sel list' db12mem04
Performing command on list..Address lookup for list.bmc failed
Could not open socket!
FAIL
Performing command on db12mem04..SEL Information
Version          : 1.5 (v1.5, v2 compliant)
Entries          : 15
Free Space       : 16144 bytes
Percent Used     : 1%
Last Add Time    : 07/04/2017 15:13:13
Last Del Time    : 06/01/2017 20:46:53
Overflow         : false
Supported Cmds   : 'Reserve'

Seems to have tried to run 'sel' on a server called 'list' ? IPMI commands can be quite long strings.

Also why have we dropped -c in favour of -k for the command argument?

**Comment by BobWhitelock** [comment]
I'll leave this for @WilliamMcCumstie to investigate as he worked on it.

But regarding `-c` changing to `-k`: we have a `--config` option now which uses `-c` as a short option, presumably that's why its been changed. I didn't realise we had a `-c` option anywhere previously though; we should probably change it back as I don't think `--config` needs a short option.

**Comment by ste78** [comment]
No need to change - that makes sense :)

**Comment by WilliamMcCumstie** [comment]
Hmmm I think the problem might be when commander parses the `cli` it strips the quotes off the command. So the following command is being ran:
`/path-to/ipmi -k sel list db12mem04`

**Comment by ColonelPanicks** [comment]
Issue testing output with @WilliamMcCumstie 
```
[root@deploy repo]# metal ipmi -k "'sel list'" db12mem04
Performing command on db12mem04..   1 | 06/01/2017 | 20:46:53 | Event Logging Disabled #0x72 | Log area reset/cleared | Asserted
   2 | 07/03/2017 | 19:27:43 | Fan #0x33 | Lower Non-critical going low  | Asserted
   3 | 07/03/2017 | 19:27:43 | Fan #0x33 | Lower Critical going low  | Asserted
   4 | 07/03/2017 | 19:28:06 | Fan #0x75 | Redundancy Lost | Asserted
   5 | 07/04/2017 | 12:29:16 | Fan #0x33 | Lower Non-critical going low  | Asserted
   6 | 07/04/2017 | 12:29:17 | Fan #0x33 | Lower Critical going low  | Asserted
   7 | 07/04/2017 | 12:29:42 | Fan #0x75 | Redundancy Lost | Asserted
   8 | 07/04/2017 | 12:41:46 | OS Boot | Installation started | Asserted
```

**Comment by WilliamMcCumstie** [comment]
Yep so that looks to be what the problem is

**Comment by ColonelPanicks** [comment]
Copied `bash_command.rb` from the `hotfix/ipmi` branch onto a running metalware install and retried ipmi command.
```
[root@deploy metalware]# metal ipmi -k 'sel list' db12mem04
Performing command on db12mem04..   1 | 06/01/2017 | 20:46:53 | Event Logging Disabled #0x72 | Log area reset/cleared | Asserted
   2 | 07/03/2017 | 19:27:43 | Fan #0x33 | Lower Non-critical going low  | Asserted
   3 | 07/03/2017 | 19:27:43 | Fan #0x33 | Lower Critical going low  | Asserted
   4 | 07/03/2017 | 19:28:06 | Fan #0x75 | Redundancy Lost | Asserted
   5 | 07/04/2017 | 12:29:16 | Fan #0x33 | Lower Non-critical going low  | Asserted
   6 | 07/04/2017 | 12:29:17 | Fan #0x33 | Lower Critical going low  | Asserted
```
yay!

**Comment by bobwhitelock** [comment]
Fixed in #105.
